Libplanet/Net/Swarm.MessageHandlers.cs

```diff
@@ -162,12 +163,15 @@ private void ProcessMessageHandler(object target, Message message)
 
         private void TransferTxs(GetTxs getTxs)
         {
-            IEnumerable<Transaction<T>> txs = getTxs.TxIds
-                .Where(txId => _store.ContainsTransaction(txId))
-                .Select(BlockChain.GetTransaction);
-
-            foreach (Transaction<T> tx in txs)
+            foreach (TxId txid in getTxs.TxIds)
             {
+                Transaction<T> tx = BlockChain.GetTransaction(txid);
+
+                if (tx is null)
```

Comment:
Can `BlockChain<T>.GetTransaction()` return `null`?

Reply:
If `txid` is invalid there's the possibility.

Reply:
🆗